pmlc/rt/xsmm.cc

```diff
@@ -34,6 +34,35 @@ extern "C" int64_t plaidml_rt_xsmm_gemm_dispatch_f32(int32_t lda, int32_t ldb,
   return reinterpret_cast<int64_t>(sgemm);
 }
 
+extern "C" void plaidml_rt_xsmm_brgemm_invoke_f32(int64_t funcAddr, float *a,
+                                                  float *b, float *c,
+                                                  int64_t numBatches) {
+  libxsmm_xmmfunction sgemm;
+  sgemm.xmm = reinterpret_cast<FunctionPtr>(funcAddr);
+  unsigned long long numBatchesVar = numBatches; // NOLINT
```

Comment:
Why is this NOLINT?

Reply:
No idea. It must have automatically gotten inserted by one of the commit hooks -- I didn't insert it myself. I am also surprised to see it.

Reply:
If you remove it, does `cpplint` complain?

Reply:
Lint doesn't like unsigned long long.

Reply:
After poking at this more, basically XSMM need unsigned long long to avoid warnings, but uint64_t (which the linter likes) is technically unsigned long on a 64 bit machine.  Which also happens to be the exact same physical type as unsigned long long on 64 bit, but they still are different logical types.  NOLINT seems the least annoying way to make things happy here.